addons/gdUnit4/src/core/GdUnitSceneRunnerImpl.gd

```diff
@@ -41,11 +41,11 @@ func _init(p_scene, p_verbose :bool, p_hide_push_errors = false):
 	set_time_factor(1)
 	# handle scene loading by resource path
 	if typeof(p_scene) == TYPE_STRING:
-		if !FileAccess.file_exists(p_scene):
+		if !ResourceLoader.exists(p_scene):
 			if not p_hide_push_errors:
-				push_error("GdUnitSceneRunner: Can't load scene by given resource path: '%s'. The resource not exists." % p_scene)
+				push_error("GdUnitSceneRunner: Can't load scene by given resource path: '%s'. The resource does not exists." % p_scene)
 			return
-		if !str(p_scene).ends_with("tscn"):
+		if !str(p_scene).ends_with(".tscn") and !str(p_scene).ends_with(".scn") and !str(p_scene).begins_with("uid://"):
```

Comment:
`ends_with(".scn")` 
From the documentation `ESCN files are compiled to binary SCN files stored inside the .godot/imported/`
I was not aware of this feature ;)
Since you added this check, can you please add a test to load a binary scene file please?